### PR TITLE
Database initialization cleanup

### PR DESCRIPTION
**Pull request by v-tec2706**

### CODE REVIEW

File: configuration.py [hunk 1]
```diff
@@ -19,4 +18,13 @@ class Configuration(Enum):
     BOT_ADDRESS = 'localhost'
 
     # -------- MONGO COLLECTIONS NAMES -----------#
+    # note: each enum name which refers to collections should contain
+    # 'collection' key word - it allows for automatic collections creation, another key word is 'capped'
```
Comment (Andrzej97) [inline configuration.py]: what is 'capped' key word use for?

File: src/common_utils/database_preparing/csv_files/DB_FINAL_100_PHRASES.csv [hunk 2]
```diff
@@ -0,0 +1,334 @@
+abwsolwentów#gospodarka#akademia#inżynier#magister#wykształcić#zapewniać#kadra#170#000#akademia wykształciła ponad 170 000 abwsolwentów - inżynierów i magistrów zapewniając kadrę dla naszej gospodarki
```
Comment (Andrzej97) [inline src/common_utils/database_preparing/csv_files/DB_FINAL_100_PHRASES.csv]: this file is no longer in use in the newest master. Please delete.

File: src/common_utils/database_preparing/csv_files/DB_FINAL_150.csv [hunk 3]
```diff
@@ -0,0 +1,150 @@
...        ,w tym także bardzo często najwyższych stanowisk                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               ,stanowią łatwy do prześledzenia praktyczny dowód jakości i użyteczności wiedzy                                                                                                                                                                                                                                                                                                                                                                                           ,jaką zdobyli oni w murach agh.
```
Comment (Andrzej97) [inline src/common_utils/database_preparing/csv_files/DB_FINAL_150.csv]: this file is no longer in use in the newest master. Please delete.

File: configuration.py (hunk 1 above)
Comment (v-tec2706) [reply configuration.py]: it is for collections which behave like cyclic buffer, or collections with a size constraints.

**Pull request merged**